src/main.ts

```
@@ -397,7 +397,8 @@
           break;
         case BackendMessageType.editCode:
           const editCodeData = data as EditCodeMessage;
-          const escapedString = editCodeData.code as string;
+          const escapedString = (editCodeData.code as string).replace(/\$/g, '\\$');
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)